.circleci/config.yml

```diff
@@ -318,6 +340,14 @@ workflows:
   commit:
     jobs:
       - compile
+      - create_dependency_backup:
+          requires:
+            - compile
+          filters:
+            tags:
+              only: /^v\d+\.\d+\.\d+$/
```

Comment:
Could you add a comment here so that it's clear what it does? Because it looks very cryptic if you have no idea what this is for.